src/main/java/racingcar/Car.java

```diff
@@ -3,13 +3,23 @@
 
 public class Car {
     private static final int MOVE_MIN_NUM = 4;
+    private final String carName;
```

Comment:
상수와 인스턴스 변수의 구분을 위해 공백 라인을 추가해보면 어떨까요?
```suggestion
    private static final int MOVE_MIN_NUM = 4;
    
    private final String carName;
```